lib/plugins/aws/remove/lib/stack.js

```diff
@@ -26,8 +27,47 @@ module.exports = {
       .then(() => cfData);
   },
 
+  disassociateUsagePlan() {
```

Comment:
@pmuens, about this. It seems that it is not possible to remove usage plan that that is associated with deployment stage using CloudFormation delete stack method. The usage plan has to be made after deployment and stage to get a reference to the stage. So delete stack tries to remove the usage plan before the stage and that cannot be done if it is associated with the stage 🤔.